### PR TITLE
Update docs for GDPR 

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

Fixes https://github.com/wso2/docs-apim/issues/1787

## Approach

- Upgraded the Versions.
- Refined and modified the command-line options of the Forget-Me tool.

## Test environment

- APIM 3.2.0

### CODE REVIEW

**Review by isharac — APPROVED** [review]

File: en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # General Data Protection Regulation (GDPR) for WSO2 API Manager
 
-The Forget-Me tool pre-packed with API-M 2.2.0 can be used to remove identities of an **external user** who is deleted according to the system administrator's request. This tool removes user identities stored in the database and also in log files in order to meet GDPR requirements. The following sections guide you through configuring and running this tool in WSO2 API Manager.
+The Forget-Me tool pre-packed with API-M 3.2.0 can be used to remove identities of an **external user** who is deleted according to the system administrator's request. This tool removes user identities stored in the database and also in log files in order to meet GDPR requirements. The following sections guide you through configuring and running this tool in WSO2 API Manager.
 
 -   [Changing the default configurations of the tool.](#GeneralDataProtectionRegulation(GDPR)forWSO2APIManager-Changingthedefaultconfigurationsofthetool.)
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md]: Do these sections have to be done in order?

File: en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md [hunk 2]
```diff
@@ -155,30 +160,30 @@ This tool is packaged with WSO2 API Manager by default. Follow the steps below t
...
 
 4.  All references to the user are removed from WSO2 API Manager. You can view the generated reports inside the `<API-M_HOME>/repository/components/tools/forget-me/conf` directory. Reports will be generated with the naming convention of `Report-<PROCESSOR>-<TIMESTAMP>.txt`  in your current working directory.
-    eg: Report-log-file-1598483873677.txt
+    Example:  `Report-log-file-1598483873677.txt`
 
-#### Running the toolkit in standalone mode
+### Running the toolkit in standalone mode
 
 This tool can run standalone and therefore cater to multiple products. This means that if you are using multiple WSO2 products and need to delete the user's identity from all products at once, you can do so by running the tool in standalone mode.
 For information on how to build and run the Forget-Me tool, see [Removing References to Deleted User Identities in WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Removing+References+to+Deleted+User+Identities+in+WSO2+Products) in the WSO2 Administration Guide.
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md:185]: This link does not seem to be applicable anymore as we are not supposed to point to the Admin guide. The content of that page seems to have been added to this page.

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md]: The order specified in the doc is correct. We need to edit the configs and config locations before running the tool .

File: en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md (hunk 2 above)
Comment (Sarangan0219) [reply en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md:185]: That doc speaks about deleting user entries from multiple wso2 products. So we need to use https://github.com/wso2/identity-anonymization-tool for that purpose and that doc specifies it.

**Review by Mariangela — APPROVED** [review]

**Pull request merged**